Client/Frontend/Browser/ContextMenuHelper.swift

```diff
@@ -126,6 +125,9 @@ extension ContextMenuHelper: TabContentScript {
         guard let data = message.body as? [String: AnyObject] else {
             return
         }
+        
+        // No image or link was pressed, we are clearing `elements` so no accidental link tap occurs.
+        if data.isEmpty { elements = nil }
```

Comment:
the context menu javascript trigger on regular taps too, so if you swipe and touch an image/link by accident, `elements` was assigned and caused a bug when context menu was showing even if you didn't tap on a link